app/src/main/AndroidManifest.xml

```diff
@@ -152,9 +161,22 @@
         <category android:name="android.intent.category.DEFAULT"/>
         <category android:name="android.intent.category.BROWSABLE"/>
       </intent-filter>
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW"/>
```

Comment:
But deep linking integration for Chapter links are maintained in `MainActivity`. We should move that to here too then.